app.libretile.Libretile.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: app.libretile.Libretile
```

Comment:
the domain `libretile.app` doesn't exist, so app-id doesn't meet the requirements.